### PR TITLE
Auto-Accept Invites To Yourself

### PR DESCRIPTION
**Pull request by sampoder**
Would resolve #4267 :D

### CODE REVIEW

File: app/models/organizer_position_invite.rb [hunk 1]
```diff
@@ -80,7 +80,11 @@ class OrganizerPositionInvite < ApplicationRecord
   after_create_commit :send_email
```
Comment (northeastprince) [inline app/models/organizer_position_invite.rb]:
```suggestion
  after_create_commit do
    user == sender ? accept : deliver
  end
```

File: app/models/organizer_position_invite.rb [hunk 2]
```diff
@@ -80,7 +80,11 @@ class OrganizerPositionInvite < ApplicationRecord
...
 
   def send_email
-    OrganizerPositionInvitesMailer.with(invite: self).notify.deliver_later
+    if self.sender.email == self.user.email
+      self.accept
+    else
+      OrganizerPositionInvitesMailer.with(invite: self).notify.deliver_later
+    end
```
Comment (northeastprince) [inline app/models/organizer_position_invite.rb]:
```suggestion
    OrganizerPositionInvitesMailer.with(invite: self).notify.deliver_later
```

File: app/models/organizer_position_invite.rb [hunk 3]
```diff
@@ -80,7 +80,11 @@ class OrganizerPositionInvite < ApplicationRecord
   after_create_commit :send_email
 
   def send_email
```
Comment (northeastprince) [inline app/models/organizer_position_invite.rb]:
```suggestion
  def deliver
```

**Comment by sampoder** [comment]
that's clean @northeastprince, nice!

**Review by YodaLightsabr — APPROVED** [review]
Looks good!

**Review by maxwofford — APPROVED** [review]
LGTM

**Pull request merged**